src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.cs

```diff
@@ -270,5 +270,15 @@ string name
 
 			return -1;
 		}
+
+		public static bool TryGetTypeByMetadataName(

```

Comment:
Created because of your comment in the last PR.  Will switch other places to use this, instead of remember null checks